analyzers/tests/SonarAnalyzer.UnitTest/TestCases/EqualityOnFloatingPoint.CSharp11.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Numerics;
+using System.Runtime.InteropServices;
+
+public class EqualityOnFloatingPoint
+{
+    bool HalfEqual(Half first, Half second)
+        => first == second; // FN
+
+    bool NFloatEqual(NFloat first, NFloat second)
+        => first == second; // FN
+
+    bool AreEqual<T>(IFloatingPointIeee754<T> first, IFloatingPointIeee754<T> second) where T : IFloatingPointIeee754<T>
+        => first == second; // FN
```

Comment:
A reminder also to myself: Since this is an FN we should open also an issue to fix.

Reply:
https://github.com/SonarSource/sonar-dotnet/issues/6423